pom.xml

```diff
@@ -41,35 +41,35 @@
 
         <product.build-time>${timestamp}</product.build-time>
 
-        <wildfly.version>18.0.1.Final</wildfly.version>
-        <wildfly.build-tools.version>1.2.12.Final</wildfly.build-tools.version>
+        <wildfly.version>19.0.0.Final</wildfly.version>
+        <wildfly.build-tools.version>1.2.13.Final</wildfly.build-tools.version>
         <eap.version>7.3.0.GA-redhat-00004</eap.version>
-        <wildfly.core.version>10.0.3.Final</wildfly.core.version>
+        <wildfly.core.version>11.0.0.Final</wildfly.core.version>
 
         <jboss.as.version>7.2.0.Final</jboss.as.version>
         <jboss.as.subsystem.test.version>7.5.22.Final-redhat-1</jboss.as.subsystem.test.version>
 
         <!-- Versions used mostly for Undertow server, aligned with WildFly -->
         <jboss.aesh.version>0.66.19</jboss.aesh.version>
         <aesh.version>2.4</aesh.version>
-        <apache.httpcomponents.version>4.5.4</apache.httpcomponents.version>
-        <apache.httpcomponents.httpcore.version>4.4.5</apache.httpcomponents.httpcore.version>
+        <apache.httpcomponents.version>4.5.11</apache.httpcomponents.version>
+        <apache.httpcomponents.httpcore.version>4.4.13</apache.httpcomponents.httpcore.version>
         <apache.mime4j.version>0.6</apache.mime4j.version>
         <jboss.dmr.version>1.5.0.Final</jboss.dmr.version>
-        <bouncycastle.version>1.60</bouncycastle.version>
-        <cxf.version>3.3.4</cxf.version>
-        <cxf.jetty.version>3.3.4</cxf.jetty.version>
-        <cxf.jaxrs.version>3.3.4</cxf.jaxrs.version>
-        <cxf.undertow.version>3.3.4</cxf.undertow.version>
+        <bouncycastle.version>1.62</bouncycastle.version>
+        <cxf.version>3.3.5</cxf.version>
+        <cxf.jetty.version>3.3.5</cxf.jetty.version>
+        <cxf.jaxrs.version>3.3.5</cxf.jaxrs.version>
+        <cxf.undertow.version>3.3.5</cxf.undertow.version>
         <dom4j.version>2.1.1</dom4j.version>
-        <github.relaxng.version>2.3.1</github.relaxng.version>
-        <h2.version>1.4.193</h2.version>
+        <github.relaxng.version>2.3.3-b02</github.relaxng.version>
+        <h2.version>1.4.197</h2.version>
         <jakarta.persistence.version>2.2.3</jakarta.persistence.version>
-        <hibernate.core.version>5.3.14.Final</hibernate.core.version>
-        <hibernate.c3p0.version>5.3.14.Final</hibernate.c3p0.version>
-        <infinispan.version>9.4.16.Final</infinispan.version>
+        <hibernate.core.version>5.3.15.Final</hibernate.core.version>
+        <hibernate.c3p0.version>5.3.15.Final</hibernate.c3p0.version>
+        <infinispan.version>9.4.18.Final</infinispan.version>
         <!-- Will be used in the product. Upstream versions are overridden in the community profile -->
-        <jackson.version>2.10.0</jackson.version>
+        <jackson.version>2.10.1</jackson.version>
```

Comment:
This version is used for product. Community version is down in community profile. Later when EAP CD19 will be out and the versions will be the same, we can simplify this.

Reply:
Done